recipes/openblas/all/conanfile.py

```diff
@@ -46,6 +50,12 @@ def _configure_cmake(self):
 
         cmake.definitions["NOFORTRAN"] = not self.options.build_lapack
```

Comment:
it would be great.

Reply:
I've updated the script to avoid unnecessary re-configuration.

While making the update, I noticed an inconsistency in the CMake build helper parameter naming: configure takes a build_folder parameter, but install takes a build_dir parameter.

Reply:
`build()` has also a `build_dir` argument. I guess it is to override `build_folder`, given as an argument to `configure()`.